src/library/js/share/share.js

```diff
@@ -535,6 +534,33 @@ export class Share {
     }
   }
 
+  /**
+   * Generate Modal for magnifying glass
+   */
+  generateModalMagnifyingGlass() {
+    const modal = $(
+      '      <div class="modal" id="modalForImg">\n' +
+        '        <div class="modal-dialog">\n' +
+        '          <div class="modal-content">\n' +
+        '            <div class="modal-header">\n' +
+        '              <span id="modalHeader"></span>\n' +
+        '              <button type="button" class="close" data-dismiss="modal">&times;</button>\n' +
+        '            </div>\n' +
+        '\n' +
+        '            <div class="modal-body">\n' +
+        '              <img src="" id="modalImg" class="imagepreview" style="width: 100%; height: 100%;" />\n' +
+        '            </div>\n' +
+        '\n' +
+        '            <div class="modal-footer">\n' +
+        '              <button type="button" class="btn btn-danger" data-dismiss="modal">Close</button>\n' +
+        '            </div>\n' +
+        '          </div>\n' +
+        '        </div>\n' +
+        '      </div>'
```

Comment:
please use multiline string with backticks
https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript

Reply:
fixed it now with multiline string